nerdamer.core.js

```diff
@@ -12465,6 +12473,33 @@ var nerdamer = (function (imports) {
         return result;
     };
 
+    /**
+     *
+     * @param {Boolean} asObject
+     * @param {String|String[]} option
+     * @returns {Array}
+     */
+    libExports.functions = function (asObject, option) {
```

Comment:
this returns list of user functions. I think it should be renamed to listFunctions or getFunctionList, maybe first one. and other things should be renamed too to keep the same pattern.